R/woods_plot.R

```diff
@@ -15,7 +15,10 @@
 #' protein identifiers.
 #' @param colouring Optional argument, column in the data frame containing information by which peptide or precursors should
 #' be coloured.
-#' @param fold_change_cutoff Optional argument specifying the log2 fold change cutoff used for assessing whether changes are significant. The default value is 2.
+#' @param fold_change_cutoff Optional argument specifying the log2 fold change cutoff used in the plot. The default value is 2.
+#' @param highlight_significant Logical, specifying whether significantly changing peptides should be highlighted with an asterisk. Default is \code{highlight_significant = FALSE}.
+#' @param significance Required if \code{highlight_significant = TRUE}. Argument specifies the column in the input data frame containing significance values (p-values, q-values).
+#' @param significance_cutoff Required if \code{highlight_significant = TRUE}. Argument specifies the significance cutoff below which peptides are considered significant.
```

Comment:
I would actually just do it the same way as colouring is provided. Would call the argument `highlight` and it should be "a logical column". So it can just contain TRUE or FALSE. If it is TRUE it makes an asterisk if not then not. That allows you to also use this feature for other things other than significance that you might want to mark (e.g. distance to a certain site). So in that case instead of setting the significance cutoff etc. here, a logical column containing the result of this cutoff should be added to the data frame beforehand.